--- postgresql-8.1.5/configure.old	2006-10-30 18:08:30.000000000 +0200
+++ postgresql-8.1.5/configure	2006-10-30 18:09:38.000000000 +0200
@@ -21993,11 +21993,11 @@
 
 
   # Note: share/locale is always the default, independent of $datadir
-  localedir='${prefix}/share/locale'
+  localedir='${prefix}/share/pgsql/locale/8.1'
   if test x"$prefix" = x"NONE"; then
-    exp_localedir="$ac_default_prefix/share/locale"
+    exp_localedir="$ac_default_prefix/share/pgsql/locale/8.1"
   else
-    exp_localedir="$prefix/share/locale"
+    exp_localedir="$prefix/share/pgsql/locale/8.1"
   fi
 
 
