--- core/src/dijkstra.c.old	2010-01-20 10:57:07.000000000 +0200
+++ core/src/dijkstra.c	2010-01-20 10:57:39.000000000 +0200
@@ -21,6 +21,7 @@
 
 #include "postgres.h"
 #include "executor/spi.h"
+#include "catalog/pg_type.h"
 #include "funcapi.h"
 
 #include "fmgr.h"
--- core/src/astar.c.old	2010-01-20 10:57:14.000000000 +0200
+++ core/src/astar.c	2010-01-20 10:57:43.000000000 +0200
@@ -21,6 +21,7 @@
 
 #include "postgres.h"
 #include "executor/spi.h"
+#include "catalog/pg_type.h"
 #include "funcapi.h"
 
 #include <stdio.h>
--- core/src/shooting_star.c.old	2010-01-20 10:57:22.000000000 +0200
+++ core/src/shooting_star.c	2010-01-20 10:57:46.000000000 +0200
@@ -21,6 +21,7 @@
 
 #include "postgres.h"
 #include "executor/spi.h"
+#include "catalog/pg_type.h"
 #include "funcapi.h"
 
 #include <stdio.h>
