--- doc/en/Makefile.in.old	2007-08-29 09:35:13.000000000 +0300
+++ doc/en/Makefile.in	2007-08-29 09:35:20.000000000 +0300
@@ -509,7 +509,6 @@
 	$(INSTALL_DATA) $(srcdir)/index.cache.bz2 $(DESTDIR)$(kde_htmldir)/$(KDE_LANG)/kpogre/; \
 	fi
 	-rm -f $(DESTDIR)$(kde_htmldir)/$(KDE_LANG)/kpogre/common
-	$(LN_S) $(kde_libs_htmldir)/$(KDE_LANG)/common $(DESTDIR)$(kde_htmldir)/$(KDE_LANG)/kpogre/common
 
 uninstall-docs:
 	-rm -rf $(DESTDIR)$(kde_htmldir)/$(KDE_LANG)/kpogre
