--- postgresql-7.4.18/configure.old	2005-01-30 21:32:19.000000000 +0200
+++ postgresql-7.4.18/configure	2005-02-23 02:22:06.000000000 +0200
@@ -13542,6 +13542,63 @@
 
 fi
 
+echo "$as_me:$LINENO: checking whether strerror_r returns int" >&5
+echo $ECHO_N "checking whether strerror_r returns int... $ECHO_C" >&6
+if test "${pgac_func_strerror_r_int+set}" = set; then
+  echo $ECHO_N "(cached) $ECHO_C" >&6
+else
+  cat >conftest.$ac_ext <<_ACEOF
+#line $LINENO "configure"
+/* confdefs.h.  */
+_ACEOF
+cat confdefs.h >>conftest.$ac_ext
+cat >>conftest.$ac_ext <<_ACEOF
+/* end confdefs.h.  */
+#include <string.h>
+int
+main ()
+{
+#ifndef _AIX
+int strerror_r(int, char *, size_t);
+#else
+/* Older AIX has 'int' for the third argument so we don't test the args. */
+int strerror_r();
+#endif
+  ;
+  return 0;
+}
+_ACEOF
+rm -f conftest.$ac_objext
+if { (eval echo "$as_me:$LINENO: \"$ac_compile\"") >&5
+  (eval $ac_compile) 2>&5
+  ac_status=$?
+  echo "$as_me:$LINENO: \$? = $ac_status" >&5
+  (exit $ac_status); } &&
+         { ac_try='test -s conftest.$ac_objext'
+  { (eval echo "$as_me:$LINENO: \"$ac_try\"") >&5
+  (eval $ac_try) 2>&5
+  ac_status=$?
+  echo "$as_me:$LINENO: \$? = $ac_status" >&5
+  (exit $ac_status); }; }; then
+  pgac_func_strerror_r_int=yes
+else
+  echo "$as_me: failed program was:" >&5
+sed 's/^/| /' conftest.$ac_ext >&5
+                                                                                                    
+pgac_func_strerror_r_int=no
+fi
+rm -f conftest.$ac_objext conftest.$ac_ext
+fi
+echo "$as_me:$LINENO: result: $pgac_func_strerror_r_int" >&5
+echo "${ECHO_T}$pgac_func_strerror_r_int" >&6
+if test x"$pgac_func_strerror_r_int" = xyes ; then
+                                                                                                    
+cat >>confdefs.h <<\_ACEOF
+#define STRERROR_R_INT
+_ACEOF
+                                                                                                    
+fi
+
 CFLAGS="$_CFLAGS"
 LIBS="$_LIBS"
 fi
