diff -Nur postgresql-9.0beta2.orig/src/backend/utils/misc/postgresql.conf.sample postgresql-9.0beta2/src/backend/utils/misc/postgresql.conf.sample
--- postgresql-9.0beta2.orig/src/backend/utils/misc/postgresql.conf.sample	2010-05-02 04:10:33.000000000 +0200
+++ postgresql-9.0beta2/src/backend/utils/misc/postgresql.conf.sample	2010-06-08 09:43:04.581995049 +0200
@@ -249,23 +249,23 @@
 
 # - Where to Log -
 
-#log_destination = 'stderr'		# Valid values are combinations of
+log_destination = 'stderr'		# Valid values are combinations of
 					# stderr, csvlog, syslog, and eventlog,
 					# depending on platform.  csvlog
 					# requires logging_collector to be on.
 
 # This is used when logging to stderr:
-#logging_collector = off		# Enable capturing of stderr and csvlog
+logging_collector = on			# Enable capturing of stderr and csvlog
 					# into log files. Required to be on for
 					# csvlogs.
 					# (change requires restart)
 
 # These are only used if logging_collector is on:
-#log_directory = 'pg_log'		# directory where log files are written,
+log_directory = 'pg_log'		# directory where log files are written,
 					# can be absolute or relative to PGDATA
-#log_filename = 'postgresql-%Y-%m-%d_%H%M%S.log'	# log file name pattern,
+log_filename = 'postgresql-%a.log'	# log file name pattern,
 					# can include strftime() escapes
-#log_truncate_on_rotation = off		# If on, an existing log file of the
+log_truncate_on_rotation = on		# If on, an existing log file of the
 					# same name as the new log file will be
 					# truncated rather than appended to.
 					# But such truncation only occurs on
@@ -273,9 +273,9 @@
 					# or size-driven rotation.  Default is
 					# off, meaning append to existing files
 					# in all cases.
-#log_rotation_age = 1d			# Automatic rotation of logfiles will
+log_rotation_age = 1d			# Automatic rotation of logfiles will
 					# happen after that time.  0 disables.
-#log_rotation_size = 10MB		# Automatic rotation of logfiles will 
+log_rotation_size = 0			# Automatic rotation of logfiles will 
 					# happen after that much log output.
 					# 0 disables.
 
