--- postgresql-7.4.25/config/python.m4.orig	2005-12-12 09:56:04.000000000 +0200
+++ postgresql-7.4.25/config/python.m4	2005-12-12 09:57:14.000000000 +0200
@@ -25,7 +25,11 @@
 python_version=`${PYTHON} -c "import sys; print sys.version[[:3]]"`
 python_prefix=`${PYTHON} -c "import sys; print sys.prefix"`
 python_execprefix=`${PYTHON} -c "import sys; print sys.exec_prefix"`
-python_configdir="${python_execprefix}/lib/python${python_version}/config"
+python_path=`${PYTHON} -c "from distutils import sysconfig; print sysconfig.get_config_vars('LIBDEST')[[0]]"`
+python_libpath=`${PYTHON} -c "from distutils import sysconfig; print sysconfig.get_config_vars('LIBDEST')[[0]][[5:]]"`
+python_configdir="${python_path}/config"
+python_moduledir="${python_path}/site-packages"
+python_moduleexecdir="${python_execprefix}/${python_libpath}/site-packages"
 python_includespec="-I${python_prefix}/include/python${python_version}"
 if test "$python_prefix" != "$python_execprefix"; then
   python_includespec="-I${python_execprefix}/include/python${python_version} $python_includespec"
@@ -37,9 +41,9 @@
 AC_SUBST(python_includespec)[]dnl
 # This should be enough of a message.
 if test "$python_prefix" != "$python_execprefix"; then
-  AC_MSG_RESULT([$python_prefix/lib/python${python_version} and $python_execprefix/lib/python${python_version}])
+  AC_MSG_RESULT([$python_prefix/${python_libpath} and $python_execprefix/${python_libpath}])
 else
-  AC_MSG_RESULT([$python_prefix/lib/python${python_version}])
+  AC_MSG_RESULT([$python_prefix/${python_libpath}])
 fi
 ])# _PGAC_CHECK_PYTHON_DIRS
 
--- postgresql-7.4.25/configure.in.orig	2005-12-12 09:57:31.000000000 +0200
+++ postgresql-7.4.25/configure.in	2005-12-12 10:01:45.000000000 +0200
@@ -19,11 +19,11 @@
 dnl
 dnl The GNU folks apparently haven't heard that some people don't use
 dnl Texinfo. Use this sorcery to use "docdir" instead of "infodir".
-m4_define([info], [doc])
-m4_define([infodir], [docdir])
+dnl
+dnl Now use both docdir and infodir to work properly with RPM macros
+dnl (courtesy, Elliott Lee <sopwith@redhat.com>)
+dnl - Andrew Overholt <overholt@redhat.com> 10 December 2002
 AC_INIT([PostgreSQL], [7.4.25], [pgsql-bugs@postgresql.org])
-m4_undefine([infodir])
-m4_undefine([info])
 AC_SUBST(docdir)
 
 AC_PREREQ(2.53)
@@ -40,6 +40,11 @@
 template=
 AC_MSG_CHECKING([which template to use])
 
+AC_ARG_WITH(docdir, [location for non-info documentation], , with_docdir=${datadir}/doc)
+docdir="$with_docdir"
+AC_SUBST(docdir)
+
+
 PGAC_ARG_REQ(with, template, [],
 [
   case $withval in
@@ -442,8 +447,8 @@
   if test -d "$krb4_prefix/include"; then
     INCLUDES="$INCLUDES -I$krb4_prefix/include"
   fi
-  if test -d "$krb4_prefix/lib"; then
-    LIBDIRS="$LIBDIRS -L$krb4_prefix/lib"
+  if test -d "$krb4_prefix/$LIBNAME"; then
+    LIBDIRS="$LIBDIRS -L$krb4_prefix/$LIBNAME"
   fi
 
   krb_srvtab="/etc/srvtab"
@@ -452,7 +457,7 @@
 
 AC_SUBST(with_krb4)
 
-
+test -z "$LIBNAME" && LIBNAME=lib
 #
 # Kerberos 5
 #
@@ -467,8 +472,8 @@
   if test -d "$krb5_prefix/include"; then
     INCLUDES="$INCLUDES -I$krb5_prefix/include"
   fi
-  if test -d "$krb5_prefix/lib"; then
-    LIBDIRS="$LIBDIRS -L$krb5_prefix/lib"
+  if test -d "$krb5_prefix/$LIBNAME"; then
+    LIBDIRS="$LIBDIRS -L$krb5_prefix/$LIBNAME"
   fi
 
   krb_srvtab="FILE:\$(sysconfdir)/krb5.keytab"
@@ -485,7 +490,7 @@
 
 AC_SUBST(krb_srvtab)
 
-
+test -z "$LIBNAME" && LIBNAME=lib
 #
 # Kerberos configuration parameters
 #
@@ -496,7 +501,7 @@
 AC_DEFINE_UNQUOTED([PG_KRB_SRVNAM], ["$with_krb_srvnam"],
                    [Define to the name of the PostgreSQL service principal in Kerberos. (--with-krb-srvnam=NAME)])
 
-
+test -z "$LIBNAME" && LIBNAME=lib
 #
 # PAM
 #
@@ -533,8 +538,8 @@
   if test -d "${openssl_prefix}/include" ; then
     INCLUDES="$INCLUDES -I${openssl_prefix}/include"
   fi
-  if test -d "${openssl_prefix}/lib" ; then
-    LIBDIRS="$LIBDIRS -L${openssl_prefix}/lib"
+  if test -d "${openssl_prefix}/$LIBNAME" ; then
+    LIBDIRS="$LIBDIRS -L${openssl_prefix}/$LIBNAME"
   fi
 ])
 
@@ -597,7 +602,7 @@
 case $host_os in sysv5*)
   AC_CACHE_CHECK([whether ld -R works], [pgac_cv_prog_ld_R],
   [
-    pgac_save_LDFLAGS=$LDFLAGS; LDFLAGS="$LDFLAGS -Wl,-R/usr/lib"
+    pgac_save_LDFLAGS=$LDFLAGS; LDFLAGS="$LDFLAGS -Wl,-R/usr/$LIBNAME"
     AC_TRY_LINK([], [], [pgac_cv_prog_ld_R=yes], [pgac_cv_prog_ld_R=no])
     LDFLAGS=$pgac_save_LDFLAGS
   ])
