--- skytools-2.1.8/sql/logtriga/logtriga.c.old	2009-03-11 17:28:41.000000000 -0700
+++ skytools-2.1.8/sql/logtriga/logtriga.c	2009-03-11 17:28:50.000000000 -0700
@@ -17,6 +17,7 @@
 #include "executor/spi.h"
 #include "commands/trigger.h"
 #include "catalog/pg_operator.h"
+#include "catalog/pg_type.h"
 #include "utils/typcache.h"
 
 #include "textbuf.h"
--- skytools-2.1.8/sql/pgq/triggers/common.c.old	2009-03-11 17:30:54.000000000 -0700
+++ skytools-2.1.8/sql/pgq/triggers/common.c	2009-03-11 17:35:34.000000000 -0700
@@ -23,10 +23,11 @@
 #include <catalog/pg_namespace.h>
 #include <executor/spi.h>
 #include <lib/stringinfo.h>
+#include <utils/builtins.h>
 #include <utils/memutils.h>
 #include <utils/inval.h>
 #include <utils/hsearch.h>
-
+#include <utils/syscache.h>
 #include "common.h"
 #include "stringutil.h"
 
