--- mammoth-replicator-8.3-1.8-beta1/src/pl/plperl/GNUmakefile.old	2008-12-29 21:02:08.000000000 +0200
+++ mammoth-replicator-8.3-1.8-beta1/src/pl/plperl/GNUmakefile	2008-12-29 21:02:27.000000000 +0200
@@ -39,6 +39,9 @@
 
 SHLIB_LINK = $(perl_embed_ldflags) $(BE_DLLLIBS)
 
+# Force rpath to be used even though we disable it everywhere else
+SHLIB_LINK += $(rpath)
+
 REGRESS_OPTS = --dbname=$(PL_TESTDB) --load-language=plperl
 REGRESS = plperl plperl_trigger plperl_shared plperl_elog
 # where to find psql for running the tests
