--- acinclude.m4.old	2008-01-19 01:45:58.000000000 -0800
+++ acinclude.m4	2008-01-19 01:45:39.000000000 -0800
@@ -228,13 +228,13 @@
 	XML2_REVISION=`expr ${XML2_VERSION} : '[0-9]*\.[0-9]*\.\([0-9]*\)'`
 	changequote([, ])dnl
 	if test "$XML2_MAJOR" -lt 2; then
-		AC_MSG_ERROR([libxml2 2.6.18 or newer is require to build pgAdmin])
+		AC_MSG_ERROR([libxml2 2.6.16 or newer is require to build pgAdmin])
 	fi
 	if test "$XML2_MAJOR" -eq 2 && test "$XML2_MINOR" -lt 6; then
-		AC_MSG_ERROR([libxml2 2.6.18 or newer is require to build pgAdmin])
+		AC_MSG_ERROR([libxml2 2.6.16 or newer is require to build pgAdmin])
 	fi
-	if test "$XML2_MAJOR" -eq 2 && test "$XML2_MINOR" -eq 6 && test "$XML2_REVISION" -lt 18; then
-		AC_MSG_ERROR([libxml2 2.6.18 or newer is require to build pgAdmin])
+	if test "$XML2_MAJOR" -eq 2 && test "$XML2_MINOR" -eq 6 && test "$XML2_REVISION" -lt 16; then
+		AC_MSG_ERROR([libxml2 2.6.16 or newer is require to build pgAdmin])
 	fi
 	AC_MSG_RESULT(ok)
 ])
--- configure.old	2008-01-19 01:45:51.000000000 -0800
+++ configure	2008-01-19 01:45:39.000000000 -0800
@@ -6160,18 +6160,18 @@
 	XML2_MINOR=`expr ${XML2_VERSION} : '[0-9]*\.\([0-9]*\)'`
 	XML2_REVISION=`expr ${XML2_VERSION} : '[0-9]*\.[0-9]*\.\([0-9]*\)'`
 		if test "$XML2_MAJOR" -lt 2; then
-		{ { echo "$as_me:$LINENO: error: libxml2 2.6.18 or newer is require to build pgAdmin" >&5
-echo "$as_me: error: libxml2 2.6.18 or newer is require to build pgAdmin" >&2;}
+		{ { echo "$as_me:$LINENO: error: libxml2 2.6.16 or newer is require to build pgAdmin" >&5
+echo "$as_me: error: libxml2 2.6.16 or newer is require to build pgAdmin" >&2;}
    { (exit 1); exit 1; }; }
 	fi
 	if test "$XML2_MAJOR" -eq 2 && test "$XML2_MINOR" -lt 6; then
-		{ { echo "$as_me:$LINENO: error: libxml2 2.6.18 or newer is require to build pgAdmin" >&5
-echo "$as_me: error: libxml2 2.6.18 or newer is require to build pgAdmin" >&2;}
+		{ { echo "$as_me:$LINENO: error: libxml2 2.6.16 or newer is require to build pgAdmin" >&5
+echo "$as_me: error: libxml2 2.6.16 or newer is require to build pgAdmin" >&2;}
    { (exit 1); exit 1; }; }
 	fi
-	if test "$XML2_MAJOR" -eq 2 && test "$XML2_MINOR" -eq 6 && test "$XML2_REVISION" -lt 18; then
-		{ { echo "$as_me:$LINENO: error: libxml2 2.6.18 or newer is require to build pgAdmin" >&5
-echo "$as_me: error: libxml2 2.6.18 or newer is require to build pgAdmin" >&2;}
+	if test "$XML2_MAJOR" -eq 2 && test "$XML2_MINOR" -eq 6 && test "$XML2_REVISION" -lt 16; then
+		{ { echo "$as_me:$LINENO: error: libxml2 2.6.16 or newer is require to build pgAdmin" >&5
+echo "$as_me: error: libxml2 2.6.16 or newer is require to build pgAdmin" >&2;}
    { (exit 1); exit 1; }; }
 	fi
 	echo "$as_me:$LINENO: result: ok" >&5
