--- skytools-2.1.7/setup.py.old	2008-12-08 00:45:33.000000000 +0200
+++ skytools-2.1.7/setup.py	2008-12-08 00:46:39.000000000 +0200
@@ -44,6 +44,7 @@
       ('share/doc/skytools/conf', [
         'python/conf/londiste.ini',
         'python/conf/pgqadm.ini',
+        'python/conf/skylog.ini',
         'python/conf/wal-master.ini',
         'python/conf/wal-slave.ini',
         'scripts/queue_mover.ini.templ',
