--- slony1-2.0.0/tools/slony-cluster-analysis-mass.sh.old	2008-12-13 01:31:02.000000000 +0200
+++ slony1-2.0.0/tools/slony-cluster-analysis-mass.sh	2008-12-13 01:31:22.000000000 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/sh
+#!/bin/sh
 # $Id: slony-cluster-analysis-mass.sh,v 1.1 2006-05-19 20:43:48 cbbrowne Exp $
 # Do cluster analyses
 
--- slony1-2.0.0/tools/slony-cluster-analysis.sh.old	2008-12-13 01:31:09.000000000 +0200
+++ slony1-2.0.0/tools/slony-cluster-analysis.sh	2008-12-13 01:31:19.000000000 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/sh
+#!/bin/sh
 # $Id: slony-cluster-analysis.sh,v 1.1 2006-05-19 20:43:48 cbbrowne Exp $
 # Analyze Slony-I Configuration
 
