diff -Nur postgresql-7.4.29.orig/config/python.m4 postgresql-7.4.29/config/python.m4
--- postgresql-7.4.29.orig/config/python.m4	2003-09-01 19:01:48.000000000 -0400
+++ postgresql-7.4.29/config/python.m4	2010-05-15 18:36:28.000000000 -0400
@@ -25,7 +25,11 @@
 python_version=`${PYTHON} -c "import sys; print sys.version[[:3]]"`
 python_prefix=`${PYTHON} -c "import sys; print sys.prefix"`
 python_execprefix=`${PYTHON} -c "import sys; print sys.exec_prefix"`
-python_configdir="${python_execprefix}/lib/python${python_version}/config"
+python_path=`${PYTHON} -c "from distutils import sysconfig; print sysconfig.get_config_vars('LIBDEST')[[0]]"`
+python_libpath=`${PYTHON} -c "from distutils import sysconfig; print sysconfig.get_config_vars('LIBDEST')[[0]][[5:]]"`
+python_configdir="${python_path}/config"
+python_moduledir="${python_path}/site-packages"
+python_moduleexecdir="${python_execprefix}/${python_libpath}/site-packages"
 python_includespec="-I${python_prefix}/include/python${python_version}"
 if test "$python_prefix" != "$python_execprefix"; then
   python_includespec="-I${python_execprefix}/include/python${python_version} $python_includespec"
@@ -37,9 +41,9 @@
 AC_SUBST(python_includespec)[]dnl
 # This should be enough of a message.
 if test "$python_prefix" != "$python_execprefix"; then
-  AC_MSG_RESULT([$python_prefix/lib/python${python_version} and $python_execprefix/lib/python${python_version}])
+  AC_MSG_RESULT([$python_prefix/${python_libpath} and $python_execprefix/${python_libpath}])
 else
-  AC_MSG_RESULT([$python_prefix/lib/python${python_version}])
+  AC_MSG_RESULT([$python_prefix/${python_libpath}])
 fi
 ])# _PGAC_CHECK_PYTHON_DIRS
 
diff -Nur postgresql-7.4.29.orig/configure.in postgresql-7.4.29/configure.in
--- postgresql-7.4.29.orig/configure.in	2010-05-13 23:44:21.000000000 -0400
+++ postgresql-7.4.29/configure.in	2010-05-15 18:43:29.000000000 -0400
@@ -19,11 +19,11 @@
 dnl
 dnl The GNU folks apparently haven't heard that some people don't use
 dnl Texinfo. Use this sorcery to use "docdir" instead of "infodir".
-m4_define([info], [doc])
-m4_define([infodir], [docdir])
+dnl
+dnl Now use both docdir and infodir to work properly with RPM macros
+dnl (courtesy, Elliott Lee <sopwith@redhat.com>)
+dnl - Andrew Overholt <overholt@redhat.com> 10 December 2002
 AC_INIT([PostgreSQL], [7.4.29], [pgsql-bugs@postgresql.org])
-m4_undefine([infodir])
-m4_undefine([info])
 AC_SUBST(docdir)
 
 m4_if(m4_defn([m4_PACKAGE_VERSION]), [2.53], [], [m4_fatal([Autoconf version 2.53 is required.
@@ -43,6 +43,10 @@
 template=
 AC_MSG_CHECKING([which template to use])
 
+AC_ARG_WITH(docdir, [location for non-info documentation], , with_docdir=${datadir}/doc)
+docdir="$with_docdir"
+AC_SUBST(docdir)
+
 PGAC_ARG_REQ(with, template, [],
 [
   case $withval in
@@ -445,8 +449,8 @@
   if test -d "$krb4_prefix/include"; then
     INCLUDES="$INCLUDES -I$krb4_prefix/include"
   fi
-  if test -d "$krb4_prefix/lib"; then
-    LIBDIRS="$LIBDIRS -L$krb4_prefix/lib"
+  if test -d "$krb4_prefix/$LIBNAME"; then
+    LIBDIRS="$LIBDIRS -L$krb4_prefix/$LIBNAME"
   fi
 
   krb_srvtab="/etc/srvtab"
@@ -455,6 +459,7 @@
 
 AC_SUBST(with_krb4)
 
+test -z "$LIBNAME" && LIBNAME=lib
 
 #
 # Kerberos 5
@@ -470,8 +475,8 @@
   if test -d "$krb5_prefix/include"; then
     INCLUDES="$INCLUDES -I$krb5_prefix/include"
   fi
-  if test -d "$krb5_prefix/lib"; then
-    LIBDIRS="$LIBDIRS -L$krb5_prefix/lib"
+  if test -d "$krb4_prefix/$LIBNAME"; then
+    LIBDIRS="$LIBDIRS -L$krb4_prefix/$LIBNAME"
   fi
 
   krb_srvtab="FILE:\$(sysconfdir)/krb5.keytab"
@@ -488,6 +493,7 @@
 
 AC_SUBST(krb_srvtab)
 
+test -z "$LIBNAME" && LIBNAME=lib
 
 #
 # Kerberos configuration parameters
@@ -499,6 +505,7 @@
 AC_DEFINE_UNQUOTED([PG_KRB_SRVNAM], ["$with_krb_srvnam"],
                    [Define to the name of the PostgreSQL service principal in Kerberos. (--with-krb-srvnam=NAME)])
 
+test -z "$LIBNAME" && LIBNAME=lib
 
 #
 # PAM
@@ -600,7 +607,7 @@
 case $host_os in sysv5*)
   AC_CACHE_CHECK([whether ld -R works], [pgac_cv_prog_ld_R],
   [
-    pgac_save_LDFLAGS=$LDFLAGS; LDFLAGS="$LDFLAGS -Wl,-R/usr/lib"
+    pgac_save_LDFLAGS=$LDFLAGS; LDFLAGS="$LDFLAGS -Wl,-R/usr/$LIBNAME"
     AC_TRY_LINK([], [], [pgac_cv_prog_ld_R=yes], [pgac_cv_prog_ld_R=no])
     LDFLAGS=$pgac_save_LDFLAGS
   ])
