--- GNUmakefile.old	2007-10-16 11:09:12.000000000 -0700
+++ GNUmakefile	2007-10-16 11:09:19.000000000 -0700
@@ -59,18 +59,6 @@
 loaderdumper-uninstall:
 	$(MAKE) -C loader uninstall
 
-templategis: Makefile.config
-	$(MAKE) -C extras/template_gis
-
-templategis-clean:
-	$(MAKE) -C extras/template_gis clean
-
-templategis-install:
-	$(MAKE) -C extras/template_gis install
-
-templategis-uninstall:
-	$(MAKE) -C extras/template_gis uninstall
-
 docs: Makefile.config
 	$(MAKE) -C doc 
 
