--- postgresql-7.4.orig/configure.in	2003-11-18 22:26:15.000000000 +0200
+++ postgresql-7.4/configure.in	2003-11-18 22:27:25.000000000 +0200
@@ -470,6 +470,9 @@
   if test -d "$krb5_prefix/include"; then
     INCLUDES="$INCLUDES -I$krb5_prefix/include"
   fi
+  if test -d "/usr/include/et"; then
+    INCLUDES="$INCLUDES -I/usr/include/et"
+  fi
   if test -d "$krb5_prefix/$LIBNAME"; then
     LIBDIRS="$LIBDIRS -L$krb5_prefix/$LIBNAME"
   fi
@@ -720,7 +723,7 @@
 fi
 
 if test "$with_krb5" = yes ; then
-  AC_SEARCH_LIBS(com_err, [krb5 'krb5 -ldes -lasn1 -lroken' com_err], [],
+  AC_SEARCH_LIBS(com_err, [com_err 'com_err' com_err], [],
                  [AC_MSG_ERROR([could not find function 'com_err' required for Kerberos 5])])
   AC_SEARCH_LIBS(krb5_encrypt, [krb5 'krb5 -ldes -lasn1 -lroken' crypto k5crypto], [],
                  [AC_MSG_ERROR([could not find function 'krb5_encrypt' required for Kerberos 5])])
