diff -Naur postgresql-8.0.0rc5.orig/src/test/regress/GNUmakefile postgresql-8.0.0rc5/src/test/regress/GNUmakefile
--- postgresql-8.0.0rc5.orig/src/test/regress/GNUmakefile	2004-11-17 13:05:06.000000000 -0500
+++ postgresql-8.0.0rc5/src/test/regress/GNUmakefile	2005-01-12 14:48:45.618011257 -0500
@@ -84,12 +84,23 @@
 testtablespace := $(abs_builddir)/testtablespace
 
 
+ifdef RPMTESTING
+define sed-command
+sed -e 's,@abs_srcdir@,$(libdir)/pgsql/test/regress,g' \
+    -e 's,@abs_builddir@,$(libdir)/pgsql/test/regress,g' \
+    -e 's,@abs_spidir@,$(libdir)/pgsql/test/regress,g' \
+    -e 's,@testtablespace@,$(libdir)/pgsql/test/regress/testtablespace,g' \
+    -e 's/@DLSUFFIX@/$(DLSUFFIX)/g' $< >$@
+endef
+else
 define sed-command
 sed -e 's,@abs_srcdir@,$(abs_srcdir),g' \
     -e 's,@abs_builddir@,$(abs_builddir),g' \
+    -e 's,@abs_spidir@,$(abs_builddir)/../../../contrib/spi,g' \
     -e 's,@testtablespace@,$(testtablespace),g' \
     -e 's/@DLSUFFIX@/$(DLSUFFIX)/g' $< >$@
 endef
+endif
 
 $(input_files): sql/%.sql: input/%.source
 	$(sed-command)
diff -Naur postgresql-8.0.0rc5.orig/src/test/regress/input/create_function_1.source postgresql-8.0.0rc5/src/test/regress/input/create_function_1.source
--- postgresql-8.0.0rc5.orig/src/test/regress/input/create_function_1.source	2002-08-21 20:01:51.000000000 -0400
+++ postgresql-8.0.0rc5/src/test/regress/input/create_function_1.source	2005-01-12 14:50:50.679124338 -0500
@@ -24,17 +24,17 @@
 
 CREATE FUNCTION check_primary_key ()
 	RETURNS trigger
-	AS '@abs_builddir@/../../../contrib/spi/refint@DLSUFFIX@'
+	AS '@abs_spidir@/refint@DLSUFFIX@'
 	LANGUAGE 'C';
 
 CREATE FUNCTION check_foreign_key ()
 	RETURNS trigger
-	AS '@abs_builddir@/../../../contrib/spi/refint@DLSUFFIX@'
+	AS '@abs_spidir@/refint@DLSUFFIX@'
 	LANGUAGE 'C';
 
 CREATE FUNCTION autoinc ()
 	RETURNS trigger
-	AS '@abs_builddir@/../../../contrib/spi/autoinc@DLSUFFIX@'
+	AS '@abs_spidir@/autoinc@DLSUFFIX@'
 	LANGUAGE 'C';
 
 CREATE FUNCTION funny_dup17 ()
diff -Naur postgresql-8.0.0rc5.orig/src/test/regress/output/create_function_1.source postgresql-8.0.0rc5/src/test/regress/output/create_function_1.source
--- postgresql-8.0.0rc5.orig/src/test/regress/output/create_function_1.source	2004-03-21 17:29:11.000000000 -0500
+++ postgresql-8.0.0rc5/src/test/regress/output/create_function_1.source	2005-01-12 14:51:08.149762180 -0500
@@ -25,15 +25,15 @@
 NOTICE:  argument type city_budget is only a shell
 CREATE FUNCTION check_primary_key ()
 	RETURNS trigger
-	AS '@abs_builddir@/../../../contrib/spi/refint@DLSUFFIX@'
+	AS '@abs_spidir@/refint@DLSUFFIX@'
 	LANGUAGE 'C';
 CREATE FUNCTION check_foreign_key ()
 	RETURNS trigger
-	AS '@abs_builddir@/../../../contrib/spi/refint@DLSUFFIX@'
+	AS '@abs_spidir@/refint@DLSUFFIX@'
 	LANGUAGE 'C';
 CREATE FUNCTION autoinc ()
 	RETURNS trigger
-	AS '@abs_builddir@/../../../contrib/spi/autoinc@DLSUFFIX@'
+	AS '@abs_spidir@/autoinc@DLSUFFIX@'
 	LANGUAGE 'C';
 CREATE FUNCTION funny_dup17 ()
         RETURNS trigger
