diff -Naur Pgtcl.orig/aclocal.m4 Pgtcl/aclocal.m4
--- Pgtcl.orig/aclocal.m4	2004-02-15 15:15:37.000000000 -0500
+++ Pgtcl/aclocal.m4	2006-01-09 12:21:32.000000000 -0500
@@ -145,7 +145,7 @@
 	    # results, and the version is kept in special file).
 	
 	    if test -r /etc/.relid -a "X`uname -n`" = "X`uname -s`" ; then
-		system=MP-RAS-`awk '{print $3}' /etc/.relid'`
+		system=MP-RAS-`awk '{print $3}' /etc/.relid`
 	    fi
 	    if test "`uname -s`" = "AIX" ; then
 		system=AIX-`uname -v`.`uname -r`
diff -Naur Pgtcl.orig/tclconfig/tcl.m4 Pgtcl/tclconfig/tcl.m4
--- Pgtcl.orig/tclconfig/tcl.m4	2004-11-11 21:55:50.000000000 -0500
+++ Pgtcl/tclconfig/tcl.m4	2006-01-09 12:22:08.000000000 -0500
@@ -752,7 +752,7 @@
 	    # results, and the version is kept in special file).
 	
 	    if test -r /etc/.relid -a "X`uname -n`" = "X`uname -s`" ; then
-		system=MP-RAS-`awk '{print $3}' /etc/.relid'`
+		system=MP-RAS-`awk '{print $3}' /etc/.relid`
 	    fi
 	    if test "`uname -s`" = "AIX" ; then
 		system=AIX-`uname -v`.`uname -r`
@@ -2012,7 +2012,7 @@
 	    # results, and the version is kept in special file).
 	
 	    if test -r /etc/.relid -a "X`uname -n`" = "X`uname -s`" ; then
-		system=MP-RAS-`awk '{print $3}' /etc/.relid'`
+		system=MP-RAS-`awk '{print $3}' /etc/.relid`
 	    fi
 	    if test "`uname -s`" = "AIX" ; then
 		system=AIX-`uname -v`.`uname -r`
