--- pgsql/contrib/intagg/int_aggregate.c	2005/03/23 19:07:03	1.11.4.3
+++ pgsql/contrib/intagg/int_aggregate.c	2005/04/23 05:39:01	1.11.4.4
@@ -222,6 +222,9 @@ int_enum(PG_FUNCTION_ARGS)
 			pc->p = p;
 			pc->flags = 0;
 		}
+		/* Now that we have a detoasted array, verify dimensions */
+		if (pc->p->a.ndim != 1)
+			elog(ERROR, "int_enum only accepts 1-D arrays");
 		pc->num = 0;
 		fcinfo->context = (Node *) pc;
 		MemoryContextSwitchTo(oldcontext);
