--- bin/Makefile.old	2008-04-14 13:08:25.000000000 +0300
+++ bin/Makefile	2008-04-14 13:08:35.000000000 +0300
@@ -22,7 +22,7 @@
 include $(top_srcdir)/contrib/contrib-global.mk
 endif
 
-pg_crc.c: $(top_srcdir)/src/backend/utils/hash/pg_crc.c
+pg_crc.c: pg_crc.c
 	rm -f $@ && $(LN_S) $< .
 
 .PHONY: subclean

--- lib/pg_btree.c.old	2008-04-18 10:31:29.000000000 +0300
+++ lib/pg_btree.c	2008-04-18 10:33:33.000000000 +0300
@@ -33,7 +33,7 @@
 #define _bt_spool			unused_bt_spool
 #define _bt_leafbuild		unused_bt_leafbuild
 
-#include "../../src/backend/access/nbtree/nbtsort.c"
+#include "nbtsort.c"
 
 #undef _bt_spoolinit
 #undef _bt_spooldestroy
--- lib/Makefile.old	2008-04-18 17:04:39.000000000 +0300
+++ lib/Makefile	2008-04-18 17:04:46.000000000 +0300
@@ -16,7 +16,7 @@
 MODULE_big = pg_bulkload
 DATA_built = pg_bulkload.sql
 DATA = uninstall_pg_bulkload.sql 
-PG_CPPFLAGS = -I../include
+PG_CPPFLAGS = -I../include -I.
 
 ifdef USE_PGXS
 PGXS := $(shell pg_config --pgxs)
