--- Makefile.old	2008-03-10 17:29:45.000000000 +0200
+++ Makefile	2008-12-02 22:21:34.000000000 +0200
@@ -9,11 +9,19 @@
 VERSION = $(shell ./pgloader.py --version |cut -d' ' -f3)
 SHORTVER= $(shell ./pgloader.py --version |cut -d' ' -f3 |cut -d '~' -f1)
 
+# for RPMs
+ifdef RPMDESTDIR
+
+libdir = $(LIBDIR)
+exdir =  $(EXDIR)
+else
+
 # debian setting
 DESTDIR =
 
 libdir   = $(DESTDIR)/usr/share/python-support/pgloader
 exdir    = $(DESTDIR)/usr/share/doc/pgloader
+endif
 
 pgloader = pgloader.py
 examples = examples
