--- postgresql-7.3.21/configure.in.old	2008-01-06 12:19:56.000000000 -0800
+++ postgresql-7.3.21/configure.in	2008-01-06 12:20:25.000000000 -0800
@@ -26,10 +26,7 @@
 m4_undefine([info])
 AC_SUBST(docdir)
 
-m4_if(m4_defn([m4_PACKAGE_VERSION]), [2.53], [], [m4_fatal([Autoconf version 2.59 is required.
-Untested combinations of 'autoconf' and PostgreSQL versions are not
-recommended.  You can remove the check from 'configure.in' but it is then
-your responsibility whether the result works or not.])])
+AC_PREREQ(2.53)
 AC_COPYRIGHT([Copyright 2002 PostgreSQL Global Development Group])
 AC_CONFIG_SRCDIR([src/backend/access/common/heaptuple.c])
 AC_CONFIG_AUX_DIR(config)
