diff -Nur postgresql-7.4.29.orig/configure.in postgresql-7.4.29/configure.in
--- postgresql-7.4.29.orig/configure.in	2010-05-15 18:54:10.000000000 -0400
+++ postgresql-7.4.29/configure.in	2010-05-15 18:53:29.000000000 -0400
@@ -475,6 +475,9 @@
   if test -d "$krb5_prefix/include"; then
     INCLUDES="$INCLUDES -I$krb5_prefix/include"
   fi
+  if test -d "/usr/include/et"; then
+    INCLUDES="$INCLUDES -I/usr/include/et"
+  fi
   if test -d "$krb4_prefix/$LIBNAME"; then
     LIBDIRS="$LIBDIRS -L$krb4_prefix/$LIBNAME"
   fi
@@ -727,7 +730,7 @@
 fi
 
 if test "$with_krb5" = yes ; then
-  AC_SEARCH_LIBS(com_err, [krb5 'krb5 -ldes -lasn1 -lroken' com_err], [],
+  AC_SEARCH_LIBS(com_err, [com_err 'com_err' com_err], [],
                  [AC_MSG_ERROR([could not find function 'com_err' required for Kerberos 5])])
   AC_SEARCH_LIBS(krb5_encrypt, [krb5 'krb5 -ldes -lasn1 -lroken' crypto k5crypto], [],
                  [AC_MSG_ERROR([could not find function 'krb5_encrypt' required for Kerberos 5])])
