--- gridsql-1.1/gridsql_env.sh.old	2009-05-12 18:52:24.000000000 +0300
+++ gridsql-1.1/gridsql_env.sh	2009-05-12 18:54:42.000000000 +0300
@@ -1,3 +1,4 @@
+#!/bin/bash
 ##########################################################################
 #
 # Copyright (C) 2008 EnterpriseDB Corporation.
@@ -15,12 +16,8 @@
 ##########################################################################
 
 # ROOT directory of where GridSQL is installed
-export GSPATH=/usr/local/gridsql-1.1
+export GSPATH=/usr
 
 # Add any referenced JDBC drivers here, like EnterpriseDB's driver
-export CLASSPATH=$GSPATH/lib/edb-jdbc14.jar:$CLASSPATH
-
-# Include scripts in user path
-export PATH=$GSPATH/bin:$PATH
-
+export CLASSPATH=/usr/share/java/edb-jdbc14.jar:$CLASSPATH
 
