diff -Naur postgresql-8.0.0rc5.orig/src/backend/utils/misc/postgresql.conf.sample postgresql-8.0.0rc5/src/backend/utils/misc/postgresql.conf.sample
--- postgresql-8.0.0rc5.orig/src/backend/utils/misc/postgresql.conf.sample	2004-11-05 14:16:16.000000000 -0500
+++ postgresql-8.0.0rc5/src/backend/utils/misc/postgresql.conf.sample	2005-01-11 19:18:21.282906899 -0500
@@ -177,22 +177,22 @@
                                 # platform.
 
 # This is relevant when logging to stderr:
-#redirect_stderr = false    # Enable capturing of stderr into log files.
+redirect_stderr = true      # Enable capturing of stderr into log files.
 # These are only relevant if redirect_stderr is true:
-#log_directory = 'pg_log'   # Directory where log files are written.
+log_directory = 'pg_log'    # Directory where log files are written.
                             # May be specified absolute or relative to PGDATA
-#log_filename = 'postgresql-%Y-%m-%d_%H%M%S.log' # Log file name pattern.
+log_filename = 'postgresql-%a.log' # Log file name pattern.
                             # May include strftime() escapes
-#log_truncate_on_rotation = false  # If true, any existing log file of the 
+log_truncate_on_rotation = true  # If true, any existing log file of the 
                             # same name as the new log file will be truncated
                             # rather than appended to.  But such truncation
                             # only occurs on time-driven rotation,
                             # not on restarts or size-driven rotation.
                             # Default is false, meaning append to existing 
                             # files in all cases.
-#log_rotation_age = 1440    # Automatic rotation of logfiles will happen after
+log_rotation_age = 1440     # Automatic rotation of logfiles will happen after
                             # so many minutes.  0 to disable.
-#log_rotation_size = 10240  # Automatic rotation of logfiles will happen after
+log_rotation_size = 0       # Automatic rotation of logfiles will happen after
                             # so many kilobytes of log output.  0 to disable.
 
 # These are relevant when logging to syslog:
