diff -Naur postgresql-8.0.0rc5.orig/src/Makefile.global.in postgresql-8.0.0rc5/src/Makefile.global.in
--- postgresql-8.0.0rc5.orig/src/Makefile.global.in	2004-12-18 21:16:18.000000000 -0500
+++ postgresql-8.0.0rc5/src/Makefile.global.in	2005-01-11 16:06:30.570550658 -0500
@@ -53,7 +53,7 @@
 # Installation directories
 #
 # These are set by the equivalent --xxxdir configure options.  We
-# append "postgresql" to some of them, if the string does not already
+# append "pgsql" to some of them, if the string does not already
 # contain "pgsql" or "postgres", in order to avoid directory clutter.
 
 prefix := @prefix@
@@ -65,21 +65,21 @@
 libexecdir := @libexecdir@
 ifeq "$(findstring pgsql, $(libexecdir))" ""
 ifeq "$(findstring postgres, $(libexecdir))" ""
-override libexecdir := $(libexecdir)/postgresql
+override libexecdir := $(libexecdir)/pgsql
 endif
 endif
 
 datadir := @datadir@
 ifeq "$(findstring pgsql, $(datadir))" ""
 ifeq "$(findstring postgres, $(datadir))" ""
-override datadir := $(datadir)/postgresql
+override datadir := $(datadir)/pgsql
 endif
 endif
 
 sysconfdir := @sysconfdir@
 ifeq "$(findstring pgsql, $(sysconfdir))" ""
 ifeq "$(findstring postgres, $(sysconfdir))" ""
-override sysconfdir := $(sysconfdir)/postgresql
+override sysconfdir := $(sysconfdir)/pgsql
 endif
 endif
 
@@ -87,7 +87,7 @@
 pkglibdir = $(libdir)
 ifeq "$(findstring pgsql, $(pkglibdir))" ""
 ifeq "$(findstring postgres, $(pkglibdir))" ""
-override pkglibdir := $(pkglibdir)/postgresql
+override pkglibdir := $(pkglibdir)/pgsql
 endif
 endif
 
@@ -95,7 +95,7 @@
 pkgincludedir = $(includedir)
 ifeq "$(findstring pgsql, $(pkgincludedir))" ""
 ifeq "$(findstring postgres, $(pkgincludedir))" ""
-override pkgincludedir := $(pkgincludedir)/postgresql
+override pkgincludedir := $(pkgincludedir)/pgsql
 endif
 endif
 includedir_server = $(pkgincludedir)/server
@@ -109,7 +109,7 @@
 ifneq (,$(docdir))
 ifeq "$(findstring pgsql, $(docdir))" ""
 ifeq "$(findstring postgres, $(docdir))" ""
-override docdir := $(docdir)/postgresql
+override docdir := $(docdir)/pgsql
 endif
 endif
 endif
