diff -Naur postgresql-7.4.5/src/backend/utils/init/miscinit.c postgresql-7.4.5-rh/src/backend/utils/init/miscinit.c
--- postgresql-7.4.5/src/backend/utils/init/miscinit.c	2003-09-26 11:27:37.000000000 -0400
+++ postgresql-7.4.5-rh/src/backend/utils/init/miscinit.c	2004-10-04 19:56:40.893167096 -0400
@@ -527,10 +527,13 @@
 		/*
 		 * Check to see if the other process still exists
 		 *
+		 * Reject parent PID too, so we are not fooled by parent shell
+		 * during reboot.
+		 *
 		 * Normally kill() will fail with ESRCH if the given PID doesn't
 		 * exist.  BeOS returns EINVAL for some silly reason, however.
 		 */
-		if (other_pid != my_pid)
+		if (other_pid != my_pid && other_pid != getppid())
 		{
 			if (kill(other_pid, 0) == 0 ||
 				(errno != ESRCH
