--- postgresql-7.4.21/configure.in.old	2008-06-11 20:34:20.000000000 +0300
+++ postgresql-7.4.21/configure.in	2008-06-11 20:35:43.000000000 +0300
@@ -26,10 +26,7 @@
 m4_undefine([info])
 AC_SUBST(docdir)
 
-m4_if(m4_defn([m4_PACKAGE_VERSION]), [2.53], [], [m4_fatal([Autoconf version 2.53 is required.
-Untested combinations of 'autoconf' and PostgreSQL versions are not
-recommended.  You can remove the check from 'configure.in' but it is then
-your responsibility whether the result works or not.])])
+AC_PREREQ(2.53)
 AC_COPYRIGHT([Copyright 2004 PostgreSQL Global Development Group])
 AC_CONFIG_SRCDIR([src/backend/access/common/heaptuple.c])
 AC_CONFIG_AUX_DIR(config)
